src/js/utils/events.js

```diff
@@ -0,0 +1,21 @@
+const validEvents = [
+	'component.render.successful',
+	'comment.posted.successful',
+	'comment.liked.successful',
+	'actions.comment.liked',
+	'auth.login.successful',
+	'auth.login.required'
+];
+
+const coralMap = new Map([
```

Comment:
We would need to add "Map" to the browserFeatures.required array in the origami.json file to ensure that consumers of this component know that they will need a Map polyfill for IE11 support.

Reply:
Good point @JakeChampion, I forgot for a moment that I am working in client side world again. How do you feel about this? I am happy to do this in a way that doesn't need polyfilling

Reply:
I think we would be happy to use Map and provide a polyfill :-)

Reply:
This is the file location you would add "Map" to by the way 👍 https://github.com/Financial-Times/o-comments/blob/master/origami.json#L15